apm-agent-plugins/apm-opentracing-plugin/src/main/java/co/elastic/apm/agent/opentracing/impl/ExternalSpanContextInstrumentation.java

```diff
@@ -121,6 +123,7 @@ public static String onExit(@Advice.FieldValue(value = "childTraceContext", typi
     @VisibleForAdvice
     @Nullable
     public static TraceContext parseTextMap(Iterable<Map.Entry<String, String>> textMap) {
+        ElasticApmTracer tracer = GlobalTracer.getTracerImpl();
```

Comment:
as previous, why not use `GlobalTracer.get()` ?

Reply:
Similar reason: because `TraceContext` needs a `ElasticApmTracer` instance so it can start spans.